docs/builders/observability-builder.md

```diff
@@ -10,6 +10,7 @@ The `ObservabilityBuilder` allows you to get started with a builder class to con
 - `enableMixedPatternAddOns`: This method helps you prepare a blueprint for setting up observability with AWS managed open source services
 - `enableOpenSourcePatternAddOns`: This method helps you prepare a blueprint for setting up observability with a combination of AWS native and AWS managed open source services
 - `enableControlPlaneLogging`: This method activates all the control plane logging features for EKS Clusters and feeds them into CloudWatch
+  - According to the [AWS documentation for Control Plane Logging](https://docs.aws.amazon.com/eks/latest/userguide/control-plane-logs.html), this is an in-place change and should work for existing and new deployments
```

Comment:
Nice call out. I would append to the same statement as function description and say. `This is an in-place change and should work for existing and new deployments, please check our [AWS documentation for Control Plane Logging](https://docs.aws.amazon.com/eks/latest/userguide/control-plane-logs.html) for more information on Control Plane logging.